### DISCUSSION TITLE
Does open3d have a function to calculate boundaries?

### DISCUSSION THREAD

**Discussion by arkerman** [isl-org/Open3D#3870] [Q&A] [answered]
BoundaryEstimation，Are there related functions?

**Comment by ssheorey** [comment, marked as answer]
You can compute the convex hull for a geometry:
http://www.open3d.org/docs/latest/python_api/open3d.geometry.TriangleMesh.html?highlight=convex%20hull#open3d.geometry.TriangleMesh.compute_convex_hull
